packages/core/src/_typography.scss

```diff
@@ -299,6 +299,7 @@ pre {
   padding: ($pt-grid-size * 1.3) ($pt-grid-size * 1.5) ($pt-grid-size * 1.2);
   line-height: 1.4;
   color: $pt-text-color;
+  font-size: $pt-grid-size * 1.3;
```

Comment:
this is like a hardcoded `smaller` so it makes sense to derive from original font size.